.github/workflows/integration_tests.yml

```diff
@@ -24,13 +24,21 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: x64
       - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+        uses: google-github-actions/setup-gcloud@v0
```

Comment:
More context here?

Reply:
Not sure we should be using a master branch here. All actions are versioned with tags, and v0 tag is updated on every new release of v0.x.y. You can see it here: https://github.com/google-github-actions/setup-gcloud/releases

Reply:
Ok, sgtm